pkg/cmd/clustertask/describe.go

```diff
@@ -112,8 +113,13 @@ NAME	STARTED	DURATION	STATUS
 {{- end }}
 `
 
+const (
+	defaultDescribeLimit = 5
```

Comment:
In the case of non TaskRun/PipelineRun resources, I wonder if a limit is necessary. For TaskRuns/PipelineRuns, it makes sense since they are ordered by start time. However, all other objects are just ordered by name. Perhaps a better approach would be to just list all available in the namespace.

Reply:
@danielhelfand , this limit is there for `logs` also. There can be lots of `pr(s)` and `tr(s)` , I think we should keep the limit and anyways we are giving an option(`--limit`) to increase it if the user wants. If you think that the default limit should not be there, then I think we should do the same for `logs` also.

Reply:
Yes, my mistake. I think the user experience around this is a bit odd, but yes you are correct. This can be kept.

Reply:
So yes, I think doing this in the case of PipelineRuns/TaskRuns makes sense for describe. Because, as you say, there are many. However, in the case of resources that are not organized in a meaningful way (i.e. ClusterTasks, Tasks, Pipelines, etc) and are bound to have fewer in a namespace, there should be no limit. With `logs`, it makes sense because those are PipelineRun/TaskRun related. There will be many so it makes sense to limit them. 

I could see later on how a filter option might help here, but I don't see limit as a good approach with non PipelineRun/TaskRun resources.

Reply:
So yes, I think doing this in the case of PipelineRuns/TaskRuns makes sense for describe. Because, as you say, there are many. However, in the case of resources that are not organized in a meaningful way (i.e. ClusterTasks, Tasks, Pipelines, etc) and are bound to have fewer in a namespace, there should be no limit. With logs, it makes sense because those are PipelineRun/TaskRun related. There will be many so it makes sense to limit them.

I could see later on how a filter option might help here, but I don't see limit as a good approach with non PipelineRun/TaskRun resources.

Logs have their own `--limit` option already set, so my opinion is it doesn't really fit here.

Reply:
Okay, I will remove the limit for non- TR/PR resources.